examples/golang/actors/http-echo-tinygo/README.md

```diff
@@ -45,13 +45,13 @@ build
 Start wasmCloud, in a separate terminal:
 
 ```console
-WASMCLOUD_VERSION=v0.79.0 wash up --nats-websocket-port 4001
+WASMCLOUD_VERSION=v0.80.0 wash up --nats-websocket-port 4001
```

Comment:
Does this work?

Reply:
Yep! I should have included this in the PR description, but here's how I tested:
- compiled wash off main
- ran `./target/debug/wash up --wasmcloud-version v0.80.0-rc3 --allow-file-load`
- ran `./target/debug/wash build` on this actor
- ran `./target/debug/wash start actor ....`
- curled the actor and confirmed it ran on the v0.80.0 host